1-select-tasks/18.txt

```diff
@@ -1,3 +1,5 @@
 Create a SQL query to calcualte total orders count by each day in 1998:
 
-| Order Date | Total Number of Orders |
\ No newline at end of file
+| Order Date | Total Number of Orders |
+
+SELECT OrderDate, COUNT(*) FROM dbo.Orders WHERE OrderDate BETWEEN '1998-01-01' AND '1998-12-31' GROUP BY OrderDate;
```

Comment:
Since the Orderdate column has the datetime type you need to convert it to date, or you might get wrong results. Also use year function to get year https://docs.microsoft.com/en-us/sql/t-sql/functions/year-transact-sql